elastic/apm-server
# Add available bulk indexers metric

## Motivation/summary

Expose the number of available bulk indexers, to see how the indexer is performing.

## Checklist




## How to test these changes

- start apm-server with `http.enabled: true`.
- curl the http endpoint and check the metric exists:
```
# curl -s localhost:5066/stats | jq .output
{
  "elasticsearch": {
    "bulk_requests": {
      "available": 10,
      "completed": 0
    }
  }
}
 ```
- ingest some events, check that the values change

## Related issues

closes #7728
